cdap-docs/integrations/source/apache-sentry.rst

```diff
@@ -8,6 +8,34 @@
 Apache Sentry
 =============
 
+.. _apache-sentry-configuration:
+
+Configuring Apache Sentry for integration with CDAP
+---------------------------------------------------
+To use CDAP on a cluster using Apache Sentry for authorization, this property needs to be set:
+
+- Add the user ``cdap`` to the Sentry property ``sentry.service.allow.connect``
+
+We also, recommend setting these properties:
+
+- Add the user ``cdap`` to the Sentry property ``sentry.service.admin.group``; this allows
```

Comment:
I understand the need of having allow connect here as its needed even when not using cdap-sentry authorization to run explore queries. But admin.group is not needed  unless you want to  do cdap authorization with sentry right ? In that case, shouldn't this just reside in sentry extension documentation ?